### PR TITLE
[Quant][fx] Fix get_default_qconfig_dict for fused modules

### PR DESCRIPTION
**Pull request by andrewor14**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #75838

Summary: Calling `prepare_fx` with `get_default_qconfig_dict`
failed for models with fused modules, such as `ConvReLU2d`.
This commit fixes this by adding qconfig entries for ReLU
and BatchNorm as well.

Test Plan:
python test/test_quantization.py TestQuantizeFx.test_qconfig_dict_with_fused_modules

Reviewers: jerryzh168

Subscribers: jerryzh168, vkuzo

Issue: https://github.com/pytorch/pytorch/issues/75825

Differential Revision: [D35668151](https://our.internmc.facebook.com/intern/diff/D35668151)

### CODE REVIEW

**Review by jerryzh168 — APPROVED** [review]
please add config for torch.nn.functional.relu and torch.relu as well

**Comment by andrewor14** [comment]
@andrewor14 has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D35668151).
